src/config/config.ts

```diff
@@ -191,7 +191,7 @@ const config = convict({
         doc: "Path to the EFS volume for storing the Git repositories",
         env: "EFS_VOL_PATH",
         format: "required-string",
-        default: "/efs/repos",
+        default: "/efs",
```

Comment:
sure, we need two repo locations. see [here](https://www.notion.so/opengov/Reduce-Amplify-Build-Times-V-851b33e935d54aaf84eaf0742d02f96f?pvs=4#66eb7a0d98e247fc87df211054cf7e4e) to know more. 
Given that is the case, our current repo path is as such 
`efs/repos`

Additional repo path to add post addition of this PR: `efs/repos-lite`

Rather than adding another env var to store staging lite branch, we keep the root the same, ie `/efs` and store the paths for the staging + staging lite repos as constants in code.

Reply:
This requires a change on SSM - pls include in PR description for on-call to know during release

Reply:
ps forgot to update, done!